gnosis/cowswap
# Disable safe for xdai

# Summary

This PR disable Gnosis Safe in xDAI network. 

For more details why:
* https://github.com/gnosis/safe-react/issues/2999
* https://github.com/gnosis/safe-react/issues/3002

It's possible Gnosis Safe team will work on a fix soon, but to reduce our risk of delaying our version we don't support Gnosis Safe for now:

![image](https://user-images.githubusercontent.com/2352112/141958112-cdc0b012-8014-4e38-9873-5ff43f6f76cf.png)

![image](https://user-images.githubusercontent.com/2352112/141958245-8a1d9de5-9e90-4b45-93df-458736553f80.png)


 # To Test

1.  Connect any wallet, or gnosis safe in Rinkeby and mainnet. Observe it still connects
2. Connect gnosis safe for xDAI. Observe it doesn't connect
